desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/OverflowingItem.tsx

```diff
@@ -15,33 +15,26 @@
 // limitations under the License.
 
 import React, { useEffect, useRef, useState } from 'react';
-import classNames from 'classnames';
 import { Button } from 'antd';
 
 import OverflowTooltip from './OverflowTooltip';
 import './OverflowingItem.scss';
 
-type OverflowingComponentType = 'breadcrumb' | 'menu';
 interface OverflowingItemProps {
   label: string;
   url: string;
-  handleFilePathChange: (path: string) => void;
-  componentType: OverflowingComponentType;
-  isLastBreadcrumb?: boolean;
+  onFilepathChange: (path: string) => void;
```

Comment:
I think we should try to keep the OverflowingItem component generic. So that it can be used by other components. That means we don't add props like `url` or custom name callbacks `onFilepathChange` etc